build-tools/merged-coverage.gradle

```diff
@@ -13,7 +13,7 @@ allprojects {
     }
 }
 
-task jacocoMerge(type: JacocoMerge) {
+/*task jacocoMerge(type: JacocoMerge) {
```

Comment:
Why are we commenting this?

Reply:
this is deprecated in `gradle 8.1.1`. https://github.com/gradle/gradle/issues/12767
https://docs.gradle.org/current/userguide/upgrading_version_7.html#jacocomerge_task_removed